network/network.go

```diff
@@ -499,3 +504,10 @@ func (n *network) nextRequestID() uint32 {
 
 	return next
 }
+
+// IsNetworkRequest checks if the given requestID is a request for this network handler (even-numbered requestIDs)
+// SDK requests are odd-numbered requestIDs
+// (see invariant: https://github.com/ava-labs/avalanchego/blob/master/network/p2p/router.go#L83)
```

Comment:
Since we are putting a line number here - we should probably not link to a branch (as the comment will become stale if the file is changed) perhaps reference a tag?